package.json

```diff
@@ -122,6 +122,9 @@
     "webpack": "4.42.1",
     "webpack-cli": "3.3.11"
   },
+  "resolutions": {
+    "@types/react": "^16.9.32"
+  },
```

Comment:
`@types/react-dom` was causing another version of `@types/react` to exist. adding this forces there to be only one version.